salt/orchestrate/aws/map_templates/edx.yml

```diff
@@ -20,5 +20,23 @@ edx:
         roles:
           - edx
           - edx-{{ app_type }}
+
+  - edx-worker-{{ environment_name }}-{{ app_type }}-{{ id_num }}:
```

Comment:
Why not use the same cloud profile? Think the only difference I noticed was that the EBS volume size was a tiny bit bigger for the edxapp. 
Also, if we use a different cloud profile, wouldn't we need to load it in edx/init.sls where we're already loading the edx.conf one?